docker/pravega/scripts/common.sh

```diff
@@ -36,9 +36,10 @@ add_system_property_ecs_config_uri() {
 
 # Add ECS certificates into Java truststore
 add_certs_into_truststore() {
-    CERTS=/etc/ssl/certs/java/ecs-certs/*
+    CERTS=/etc/secret-volume/ca-bundle/*
     for cert in $CERTS
     do
       yes | keytool -importcert -storepass changeit -file "${cert}" -keystore /etc/ssl/certs/java/cacerts || true
     done
+    rm -rf /etc/secret-volume/ca-bundle/* || true
```

Comment:
This is unrequired and potentially dangerous.  If Pravega is running bare metal then this will delete any certificates that were placed in `/etc/secret-volume/ca-bundle`.  If this is running in Kubernetes then it would be a secret and you can't delete the certificates from the secret.

Is there a reason this was added?

Reply:
This is per our security specialist Ravi's comment.
Functional-wise, it's tested okay to delete certificates after adding them into truststore. Those certificates have been mounted into the pod already so it wont affect k8s secrets.

Reply:
Actually, @maddisondavid is right. You shouldn't delete a file-mounted K8s secret using an OS command. If you manage to do it, it'll probably automatically be recreated (I am not 100% sure). Maybe we should leave it as is as @maddisondavid suggests.

Reply:
fixed